cavern/src/main/java/org/opencadc/cavern/nodes/FileSystemNodePersistence.java

```diff
@@ -183,15 +182,16 @@ public FileSystemNodePersistence() {
         // only require a group mapper because IVOA GMS does not include numeric gid
         // assume user mapper is the same service
         URI posixMapperID = la.getServiceURI(Standards.POSIX_GROUPMAP.toASCIIString());
+        PosixMapperClient posixMapper;
         if ("https".equals(posixMapperID.getScheme())) {
             try {
                 URL baseURL = posixMapperID.toURL();
-                this.posixMapper = new MyPosixMapperClient(baseURL);
+                posixMapper = new MyPosixMapperClient(baseURL);
             } catch (MalformedURLException ex) {
                 throw new InvalidConfigException("invalid " + Standards.POSIX_GROUPMAP.toASCIIString() + " base URL: " + posixMapperID, ex);
             }
         } else {
-            this.posixMapper = new MyPosixMapperClient(posixMapperID);
+            posixMapper = new MyPosixMapperClient(posixMapperID);
```

Comment:
prefer to keep the `this.` prefix on the left hand side when assigning to member vars; it stands out when a bit better in a non-syntax highlighting environment

Reply:
The `posixMapper` variable was moved into a local variable as it was only used in creating the `GroupCache` object.  It wasn't used as a Class member.

Reply:
ah, ok. makes sense.